vm/src/stdlib/itertools.rs

```diff
@@ -174,14 +173,14 @@ mod decl {
     #[pyclass(name = "count")]
     #[derive(Debug, PyPayload)]
     struct PyItertoolsCount {
-        cur: PyRwLock<BigInt>,
-        step: BigInt,
+        cur: PyRwLock<PyObjectRef>,
+        step: Option<PyIntRef>,
```

Comment:
Can I ask you is there any reason for making `step` as `Option`?
It seems `step` also need to be checked via `PyNumber::check` in cpython which means that any type that implements `number protocol` can be fitted.

Reply:
```suggestion
        step: PyObjectRef,
```

Reply:
Because step is an optional value as the second argument of count, set it as Option.
And when processing, there is something similar to repeat, so I considered it.
But if I need to set it up differently, I will.
I checked all reviews. thank. I'll fix.

Reply:
What's different between saving it as a `PyIntRef` and `Option<PyIntRef>`?
e.g. Does it affect to `repr` result? If they act same for everything, we don't need to distinguish them.